src/components/Dashboard/index.js

```diff
@@ -0,0 +1,42 @@
+import React, {Fragment} from 'react';
+import PostsContainer from  './PostsContainer';
+import Header from '../Header';
+import SiteStats from "./SiteStats";
+import SupportRequests from "./SupportRequests";
+import NewsContainer from "./NewsContainer";
+
+const DashboardContext = React.createContext();
+
+class DashboardProvider extends React.Component {
+	state = {}
```

Comment:
Ah is this the react alpha version thing?

Reply:
yeah, but probably not necessary, will maybe remove before finishing PR